client/src/components/MemberCard/Card.jsx

```diff
@@ -21,7 +23,13 @@ const Card = (props) => {
                 <h3 className="text-2xl text-slate-700 font-bold leading-normal mb-1">{props.data.name}</h3>
                 <div className="text-s mt-0 mb-2 text-slate-400 font-bold">
                     <i className="fas fa-map-marker-alt mr-2 text-slate-500 opacity-75">{props.data.year}</i>
-                    
+                    <div>
+                        <h3 className="rounded-lg bg-red-100 text-pink-600 font-bold mb-1" style={{ display: 'inline-block', marginRight: '5px' }}>{props.infoMl}</h3>
+                        <h3 className="rounded-lg bg-gray-200 text-purple-600 font-bold mb-1" style={{ display: 'inline-block', marginRight: '5px' }}>{props.infoApi}</h3>
+                        <h3 className="rounded-lg bg-indigo-100 text-blue-600 font-bold mb-1" style={{ display: 'inline-block', marginRight: '5px' }}>{props.infoReact}</h3>
+                        <h3 className="rounded-lg bg-green-200 text-green-700 font-bold mb-1" style={{ display: 'inline-block' }}>{props.infoTeamLead}</h3>
```

Comment:
Your choice of props is good. But I would suggest tweaking the way you're using them. The main thing that stuck out to me was the amount of code repetition, and that is because you're using the props to explicitly write out the team cards for various combinations of isXTeam props. This isn't scalable. Instead of wrapping the entire card output in different if conditions like in line 11, write the outer card output once (around lines 57-90) and wrap the `<h3>` tags in if conditions since that's the only output that's changing. That way you only render that specific `<h3>` tag (ex: `<h3>` tag for ML) if the `isMlTeam` prop is true, and render nothing/null if false. A more explicit example would look something like this: 
```
if(props.isMlTeam){
  <h3 className="..." style={{...}}>{props.infoMl}</h3>
}
```
You'll use a similar format for the other three team card props, too. This is what conditional rendering refers to. React has different solutions to this, and I suggest taking a look at this section of the React docs to learn from different examples: https://react.dev/learn/conditional-rendering. You could even take the code above and make it it's own component that you can then pass those team specific props into. The first thing I would do is remove the if conditionals that you already have, and then move on with the conditional rendering stuff above.